### PR TITLE
support multiple 3.x pause tags and set 3.4.1 for 1.20

### PR DESCRIPTION
**Pull request by jaxesn**
kubeadm 1.21 hardcodes 3.4.1 as the pause tag, we need to push this in addition to 3.4 for 1.20.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by abhay-krishna — APPROVED** [review]
/lgtm

File: projects/kubernetes/kubernetes/Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ RELEASE?=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/$(RELEASE_ENVIR
 ARTIFACT_BUCKET?=my-s3-bucket
 GIT_TAG?=$(shell cat ./$(RELEASE_BRANCH)/GIT_TAG)
 GOLANG_VERSION?=$(shell cat ./$(RELEASE_BRANCH)/GOLANG_VERSION)
-PAUSE_TAG?=$(shell cat ./$(RELEASE_BRANCH)/PAUSE_TAG)
+PAUSE_TAG?=$(shell cat $(wildcard ./$(RELEASE_BRANCH)/PAUSE_*TAG))
```
Comment (TerryHowe) [inline projects/kubernetes/kubernetes/Makefile:8]: I don't have this command on my mac

Comment (abhay-krishna) [reply projects/kubernetes/kubernetes/Makefile:8]: This is a Make function, so it will be recognized only within the scope of the Makefile.

Comment (TerryHowe) [reply projects/kubernetes/kubernetes/Makefile:8]: Thanks. Make is so weird

**Pull request merged**